colossalai/zero/init_ctx/init_context.py

```diff
@@ -88,6 +88,8 @@ class ZeroContextConfig(object):
     """The configuration used to control zero context initialization.
 
     Args:
+        is_replicated (bool, optional): Is param replicated across data parallel group.
```

Comment:
raning to use_replicated.
Do not use is_xxx as args

Reply:
We do not use the parameters as replicated. 
But the parameters are replicated inherently.

Reply:
Maybe just use 'replicated' instead.